Dolphin scripts/Entrance Randomizer/lib/constants.py

```diff
@@ -157,26 +157,33 @@ class Addresses:
 
 class LevelCRC(IntEnum):
     JAGUAR = 0x99885996
-    CRASH_SITE = 0xEE8F6900
-    PLANE_COCKPIT = 0x4A3E4058
-    FLOODED_COURTYARD = 0xECC9D759
-    CHAMELEON_TEMPLE = 0x0081082C
+    PLANE_CUTSCENE = 0x53257119
     JUNGLE_CANYON = 0xDEDA69BC
-    MAMA_OULLO_TOWER = 0x07ECCC35
-    VIRACOCHA_MONOLITHS = 0x6F498BBD
-    VIRACOCHA_MONOLITHS_CUTSCENE = 0xE8362F5F
-    ALTAR_OF_AGES = 0xABD7CCD8
+    PLANE_COCKPIT = 0x4A3E4058
     BITTENBINDERS_CAMP = 0x0EF63551
-    MYSTERIOUS_TEMPLE = 0x099BF148
+    MOUTH_OF_INTI = 0xB8D5CE86
+    FLOODED_COURTYARD = 0xECC9D759
+    TURTLE_MONUMENT = 0x239A2165
+    NATIVE_VILLAGE = 0x05AA726C
+    RENEGADE_HEADQUARTERS = 0x1CB1432D
     EYES_OF_DOOM = 0x9A0C8DF8
     SCORPION_TEMPLE = 0x4B08BBEB
-    PENGUIN_TEMPLE = 0x1B11EC74
+    SCORPION_SPIRIT = 0x0305DC42
+    MOUNTAIN_OVERLOOK = 0x907C492B
+    WHITE_VALLEY = 0x62548B77
+    CAVERN_LAKE = 0x8B372E42
+    VIRACOCHA_MONOLITHS = 0x6F498BBD
+    VIRACOCHA_MONOLITHS_CUTSCENE = 0xE8362F5F
     VALLEY_OF_THE_SPIRITS = 0x08E3C641
     COPACANTI_LAKE = 0x8147EA91
+    MOUNTAIN_SLED_RUN = 0x1B8833D3
     APU_ILLAPU_SHRINE = 0x5511C46C
+    MYSTERIOUS_TEMPLE = 0x099BF148
+    ALTAR_OF_AGES = 0xABD7CCD8
+    ST_CLAIRE_DAY = 0xBA9370DF
     ST_CLAIRE_NIGHT = 0x72AD42FA
-    ST_CLAIRE_DAY = 0x72AD42FA
-    TELEPORTS = 0xE97CB47C
+    GATES_OF_EL_DORADO = 0x93F89D45
+    PUSCA = 0xCFD2FE10
```

Comment:
I chose the order of levels listed that made the most sense to me.
If you want to argue another order is better I'm willing to hear you out.

The values I removed were values that were not referenced anywhere in the code, and I'm told best practise is to never create variables (or enum values) that never get referenced.
Once again I'm up for discussion of you want.

Reply:
> best practise is to never create variables (or enum values) that never get referenced.

Sure you can remove "dead code"

But this has a known finite list of values. and there's a very high chance for which values being used currently in code to change based on the features and workarounds we add. Adding and removing constantly would lead to more churn. It's also useful to already have the complete enum\* if you suddenly need to reference one of the levels directly.

\* The enum currently isn't complete, but as mentioned that was more due to lazyness.

Reply:
All levels are now added to enum, and are listed in alphabetical order.